.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,10 +1,10 @@
 name: Feature request
 description: Suggest an idea for this project
-title: 'auro-formvalidation: [feature summary]'
+title: 'wc-generator: [feature summary]'
```

Comment:
revert

Reply:
This is your commit from March 8th https://github.com/AlaskaAirlines/auro-formvalidation/commit/fd17eb91165c1c79d8944b813b40601495fd2945

You changed `auro-formvalidation` to `wc-generator`. I assume that you copied this template from the generator and forgot to change this back?